### PR TITLE
fixed errors

### PR DESCRIPTION
**Pull request by LizzAlice**
fixed issue with previous commit

### CODE REVIEW

File: mail/forms.py [hunk 1]
```diff
@@ -21,7 +21,11 @@ class MailTemplateUpdateForm(MailTemplateForm):
     def __init__(self, *args, **kwargs):
         super(MailTemplateForm, self).__init__(*args, **kwargs)
         if self.instance.department:
-            used = MailTemplate.objects.values_list('usage', flat=True).filter(department = kwargs["instance"].department)
-            used = [x for x in used if str(self.instance.usage) not in x]
+            used_db = MailTemplate.objects.values_list('usage', flat=True).filter(department = kwargs["instance"].department)
+            used = []
+            for x in used_db:
+                if x:
```
Comment (octomike) [inline mail/forms.py]: always check for None with `if x is not None`

File: mail/forms.py [hunk 2]
```diff
@@ -21,7 +21,11 @@ class MailTemplateUpdateForm(MailTemplateForm):
...
+                    if str(self.instance.usage) not in x:
```
Comment (octomike) [inline mail/forms.py]: both operands are a string here, right?

File: mail/forms.py (hunk 1 above)
Comment (LizzAlice) [reply mail/forms.py]: why? I thought if x: did the same thing?

File: mail/forms.py (hunk 2 above)
Comment (LizzAlice) [reply mail/forms.py]: None is not

File: mail/forms.py (hunk 1 above)
Comment (LizzAlice) [reply mail/forms.py]: ok, I forgot the false case

**Pull request merged**